examples/source/visual-effects/Ken_Burns.html

```diff
@@ -63,12 +63,14 @@
     - Animate the `<amg-img>` element using the `animate-in="pan-right"`. (You can also use `pan-left`, `pan-up`, and `pan-down`). [See amp-story animations list.](/documentation/examples/visual-effects/amp_story_animations/?format=stories#zoom-in)
 
     Tip: It's important that the **image** sets the **panning animation**, whereas the image **container** sets the **zoom animation**. Otherwise the dimensions of the image will change, causing undesired results in the animation.
+    
+    Tip: Animations can help make your Web Story more visually exciting and engaging, but use them sparingly. Some users may find long, continuous animations distracting. Other users may have motion sensitivity and be adversely affected by excessive use of motion and parallax effects.
```

Comment:
Can we add a link to ampproject/amphtml#24821, which tracks respecting the prefers-reduced-motion directive for such users?

Reply:
as far as i can tell, these animation effects already respect that setting. but not all users will have that setting always turned on, and this is still good advice in general about not overdoing it in general, I'd say.